podcasts/TranscriptSearchAccessoryView.swift

```diff
@@ -63,6 +63,11 @@ class TranscriptSearchAccessoryView: UIInputView {
         contentInsets: .init(top: 8, leading: 8, bottom: 8, trailing: 8)
     )
 
+    func enableUpDownButtons(_ enable: Bool) {
```

Comment:
It was intentional to not do this because the search on other parts of iOS doesn't handle disabling the buttons.

I don't think it's bad having though.

However, when I first open the search the buttons are enabled, they should be disabled if we want this behavior to be correct.

Reply:
Updated here: [68a71f3](https://github.com/Automattic/pocket-casts-ios/pull/2000/commits/68a71f30977360d288e50254655404e124231845)